modules/Bio/EnsEMBL/Compara/Production/Analysis/AlignmentChains.pm

```diff
@@ -84,12 +65,6 @@ sub run_chains {
   # write the query in nib format 
   # for use by lavToAxt;
   #################################
-  if ($self->param('query_nib_dir')) {
```

Comment:
I removed it because this parameter is not included (not even as a comment) in the pipeconfig file, so I thought it belonged to a deprecated code. I will put this and the code below back in place.